patches/000-initial-setup.patch

```diff
@@ -633,28 +759,29 @@ index 69c509a771..05eac35d88 100644
  	pub := &PublicKey{
  		E: 65537,
 diff --git a/src/crypto/rsa/pss_test.go b/src/crypto/rsa/pss_test.go
-index 51f9760187..93bbc26bd0 100644
+index 51f9760187..ad82d4456f 100644
 --- a/src/crypto/rsa/pss_test.go
 +++ b/src/crypto/rsa/pss_test.go
-@@ -9,6 +9,7 @@ import (
+@@ -9,6 +9,8 @@ import (
  	"bytes"
  	"compress/bzip2"
  	"crypto"
 +	"crypto/internal/boring"
++	"crypto/internal/backend/boringtest"
  	"crypto/rand"
  	"crypto/sha1"
  	"crypto/sha256"
-@@ -76,6 +77,9 @@ func TestEMSAPSS(t *testing.T) {
+@@ -76,6 +78,9 @@ func TestEMSAPSS(t *testing.T) {
  // TestPSSGolden tests all the test vectors in pss-vect.txt from
  // ftp://ftp.rsasecurity.com/pub/pkcs/pkcs-1/pkcs-1v2-1-vec.zip
  func TestPSSGolden(t *testing.T) {
-+	if boring.Enabled {
++	if boring.Enabled && ! boringtest.Supports(t, "SHA1") {
```

Comment:
nit: Do you mind cleaning up the space in the negation operator? E.g. `!assertion()` as opposed to `! assertion()`.